File: Iniciante/1013/1013.py
float_str = input()
a, b, c = float_str.split()
a = float(a)
b = float(b)
c = float(c)

maiorAB = ((a + b + abs(a - b)) / 2)

if maiorAB > c:
    print("%d eh o maior" %maiorAB)
else:
    print("%d eh o maior" %c)

File: Iniciante/Pagina 1/1013/1013.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int A, B, C, maiorAB;

    scanf("%d", &A); 
    scanf("%d", &B); 
    scanf("%d", &C);

    //"abs() mostra o valor modular da operaçao"
    maiorAB = ((A + B + abs(A - B)) / 2);

    if (maiorAB > C) {
        printf("%d eh o maior\n", maiorAB);
    } else {
        printf("%d eh o maior\n", C);
    }

}